test/unit/api/cpp/api_types_black.cpp

```diff
@@ -59,8 +60,8 @@ TEST_F(TestApiTypes, printEnum)
     ss << std::to_string(cvc5::modes::ProofComponent::FULL) << " ";
     ss << std::to_string(cvc5::modes::FindSynthTarget::ENUM) << " ";
     ss << std::to_string(cvc5::modes::InputLanguage::SMT_LIB_2_6) << " ";
-    ss << std::to_string(cvc5::modes::ProofFormat::LFSC);
-    ASSERT_EQ(ss.str(), expected);
```

Comment:
I believe you accidentally removed this line. Without it, you are not testing the last chunk of code.